test/parallel/test-tick-processor-builtin.js

```diff
@@ -1,19 +1,11 @@
 'use strict';
 const common = require('../common');
-const path = require('path');
 
-// TODO(mhdawson) Currently the test-tick-processor functionality in V8
-// depends on addresses being smaller than a full 64 bits.  Aix supports
-// the full 64 bits and the result is that it does not process the
-// addresses correctly and runs out of memory
-// Disabling until we get a fix upstreamed into V8
-if (common.isAix) {
```

Comment:
Sorry, I think I missed a refactoring CL in here. Why is this check no longer necessary?